Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM centos:centos8
```

Comment:
Out of curiosity, why does this project use centos? Seems like everything else is debian/ubuntu based...

Reply:
I think it was just the path of least resistence to get the scheduler dependencies running since the scheduler originally ran on a Centos machine out of docker. It's probably not necessary, but it might take a bit of work to get all the dependencies correct on another OS...